src/Schema/ContextModule.ts

```diff
@@ -12,17 +13,21 @@ export enum ContextModule {
   artworkImage = "artworkImage",
   artworkSidebar = "artworkSidebar",
   auctionSidebar = "auctionSidebar",
+  auctionCail = "auctionCard",
```

Comment:
@mikehrom is this supposed to be "auctionCard"?

Reply:
Yes fixed - thank you 👍